eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php

```diff
@@ -53,4 +64,27 @@ public function getFieldErrors()
     {
         return $this->errors;
     }
+
+    private function generateValidationErrorsMessages(): string
+    {
+        $message = '';
+        foreach ($this->getFieldErrors() as $validationErrors) {
+            foreach ($validationErrors as $validationError) {
+                if (is_array($validationError)) {
+                    foreach ($validationError as $item) {
+                        $message .= $this->generateValidationErrorMessage($item);
+                    }
+                } else {
+                    $message .= $this->generateValidationErrorMessage($validationError);
+                }
```

Comment:
What is the use case for this structure to be object instead of sub-array? AFAIK this is always map of Field Defintion to Language Code (or the other way around) to the error message.

Reply:
This can be either an object or an array, depending on the number of validation errors for a single field identifier.